src/benchmark_api_internal.h

```diff
@@ -2,13 +2,22 @@
 #define BENCHMARK_API_INTERNAL_H
 
 #include "benchmark/benchmark.h"
+#include "commandlineflags.h"
 
 #include <cmath>
 #include <iosfwd>
 #include <limits>
 #include <string>
 #include <vector>
 
+DECLARE_double(benchmark_min_time);
```

Comment:
why here and not benchmark_runner.h?

Reply:
Ah, putting them into `benchmark_runner.h` works too.
But specifically into header, *not* the `.cc`!